*** ./smali/android/widget/RemoteViews.smali	2012-10-01 10:39:08.991304019 +0800
--- /home/crazyi/android/patchrom/ef39s/temp/new_smali/framework.jar.out/./smali/android/widget/RemoteViews.smali	2012-10-01 10:40:30.675301225 +0800
***************
*** 65,70 ****
--- 65,72 ----
  
  .field private final mPackage:Ljava/lang/String;
  
+ .field mDefaultTheme:I
+ 
  
  # direct methods
  .method static constructor <clinit>()V
***************
*** 445,450 ****
--- 447,453 ----
      move-result-object v0
  
      .local v0, c:Landroid/content/Context;
+    
      :goto_0
      return-object v0
  
***************
*** 570,575 ****
--- 573,583 ----
      move-result-object v0
  
      .local v0, c:Landroid/content/Context;
+ 
+     iget v3, p0, Landroid/widget/RemoteViews;->mDefaultTheme:I
+ 
+     invoke-virtual {v0, v3}, Landroid/content/Context;->setTheme(I)V
+     
      const-string v3, "layout_inflater"
  
      invoke-virtual {v0, v3}, Landroid/content/Context;->getSystemService(Ljava/lang/String;)Ljava/lang/Object;
***************
*** 1538,1540 ****
--- 1546,1561 ----
      :cond_2
      return-void
  .end method
+ 
+ .method public setDefaultTheme(I)V
+     .locals 0
+     .parameter "id"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->NEW_METHOD:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
+ 
+     .prologue
+     iput p1, p0, Landroid/widget/RemoteViews;->mDefaultTheme:I
+ 
+     return-void
+ .end method
