*** ./smali/android/widget/TextView.smali	2012-10-01 10:39:07.411304073 +0800
--- /home/crazyi/android/patchrom/ef39s/temp/new_smali/framework.jar.out/./smali/android/widget/TextView.smali	2012-10-01 10:40:27.935301319 +0800
***************
*** 29143,29145 ****
--- 29143,29181 ----
      :cond_0
      return-void
  .end method
+ 
+ .method public setCursorDrawableRes(I)V
+     .locals 3
+     .parameter "cursorDrawableRes"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->NEW_METHOD:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
+ 
+     .prologue
+     iput p1, p0, Landroid/widget/TextView;->mCursorDrawableRes:I
+ 
+     const/4 v0, 0x0
+ 
+     .local v0, i:I
+     :goto_0
+     iget v1, p0, Landroid/widget/TextView;->mCursorCount:I
+ 
+     if-ge v0, v1, :cond_0
+ 
+     iget-object v1, p0, Landroid/widget/TextView;->mCursorDrawable:[Landroid/graphics/drawable/Drawable;
+ 
+     const/4 v2, 0x0
+ 
+     aput-object v2, v1, v0
+ 
+     add-int/lit8 v0, v0, 0x1
+ 
+     goto :goto_0
+ 
+     :cond_0
+     const/4 v1, 0x0
+ 
+     iput v1, p0, Landroid/widget/TextView;->mCursorCount:I
+ 
+     return-void
+ .end method
\ 文件尾没有 newline 字符
