*** ./smali/android/widget/TimePicker$5.smali	2012-10-01 10:39:06.519304104 +0800
--- /home/crazyi/android/patchrom/ef39s/temp/new_smali/framework.jar.out/./smali/android/widget/TimePicker$5.smali	2012-10-01 10:40:26.475301369 +0800
***************
*** 72,77 ****
--- 72,81 ----
      #calls: Landroid/widget/TimePicker;->updateAmPmControl()V
      invoke-static {v0}, Landroid/widget/TimePicker;->access$200(Landroid/widget/TimePicker;)V
  
+     iget-object v0, p0, Landroid/widget/TimePicker$5;->this$0:Landroid/widget/TimePicker;
+ 
+     invoke-virtual {v0}, Landroid/widget/TimePicker;->onTimeChanged()V
+ 
      return-void
  
      :cond_0
