*** ./smali/android/content/res/Resources$Theme.smali	2012-10-01 10:39:18.811303684 +0800
--- /home/crazyi/android/patchrom/ef39s/temp/new_smali/framework.jar.out/./smali/android/content/res/Resources$Theme.smali	2012-10-01 10:40:45.063300734 +0800
***************
*** 1,4 ****
! .class public final Landroid/content/res/Resources$Theme;
  .super Ljava/lang/Object;
  .source "Resources.java"
  
--- 1,4 ----
! .class public Landroid/content/res/Resources$Theme;
  .super Ljava/lang/Object;
  .source "Resources.java"
  
***************
*** 9,15 ****
  .end annotation
  
  .annotation system Ldalvik/annotation/InnerClass;
!     accessFlags = 0x11
      name = "Theme"
  .end annotation
  
--- 9,15 ----
  .end annotation
  
  .annotation system Ldalvik/annotation/InnerClass;
!     accessFlags = 0x1
      name = "Theme"
  .end annotation
  
