*** ./smali/com/android/server/AppWidgetService$3.smali	2012-10-01 10:39:54.091302476 +0800
--- /home/crazyi/android/patchrom/ef39s/temp/new_smali/services.jar.out/./smali/com/android/server/AppWidgetService$3.smali	2012-10-01 10:41:24.787299374 +0800
***************
*** 63,68 ****
--- 63,95 ----
      return-void
  
      :cond_1
+     const-string v15, "android.intent.action.RESTORE_FINISH"
+ 
+     invoke-virtual {v15, v2}, Ljava/lang/String;->equals(Ljava/lang/Object;)Z
+ 
+     move-result v15
+ 
+     if-eqz v15, :cond_z
+ 
+     move-object/from16 v0, p0
+ 
+     iget-object v15, v0, Lcom/android/server/AppWidgetService$3;->this$0:Lcom/android/server/AppWidgetService;
+ 
+     const/16 v16, 0x0
+ 
+     move/from16 v0, v16
+ 
+     iput-boolean v0, v15, Lcom/android/server/AppWidgetService;->mStateLoaded:Z
+ 
+     move-object/from16 v0, p0
+ 
+     iget-object v15, v0, Lcom/android/server/AppWidgetService$3;->this$0:Lcom/android/server/AppWidgetService;
+ 
+     invoke-virtual {v15}, Lcom/android/server/AppWidgetService;->sendInitialBroadcasts()V
+ 
+     goto :goto_0
+ 
+     :cond_z
      const-string v15, "android.intent.action.CONFIGURATION_CHANGED"
  
      invoke-virtual {v15, v2}, Ljava/lang/String;->equals(Ljava/lang/Object;)Z
