*** ./smali/com/android/server/AppWidgetService.smali	2012-10-01 10:39:52.127302544 +0800
--- /home/crazyi/android/patchrom/ef39s/temp/new_smali/services.jar.out/./smali/com/android/server/AppWidgetService.smali	2012-10-01 10:41:22.191299462 +0800
***************
*** 5318,5323 ****
--- 5318,5335 ----
      monitor-exit v3
      :try_end_0
      .catchall {:try_start_0 .. :try_end_0} :catchall_0
+     
+     iget-object v2, p0, Lcom/android/server/AppWidgetService;->mContext:Landroid/content/Context;
+ 
+     iget-object v3, p0, Lcom/android/server/AppWidgetService;->mBroadcastReceiver:Landroid/content/BroadcastReceiver;
+ 
+     new-instance v4, Landroid/content/IntentFilter;
+ 
+     const-string v5, "android.intent.action.RESTORE_FINISH"
+ 
+     invoke-direct {v4, v5}, Landroid/content/IntentFilter;-><init>(Ljava/lang/String;)V
+ 
+     invoke-virtual {v2, v3, v4, v6, v6}, Landroid/content/Context;->registerReceiver(Landroid/content/BroadcastReceiver;Landroid/content/IntentFilter;Ljava/lang/String;Landroid/os/Handler;)Landroid/content/Intent;
  
      iget-object v2, p0, Lcom/android/server/AppWidgetService;->mContext:Landroid/content/Context;
  
